superset-frontend/src/views/components/Menu.tsx

```diff
@@ -185,6 +185,24 @@ export function Menu({
   const screens = useBreakpoint();
   const uiConig = useUiConfig();
 
+  const nonItemsToRender = {
```

Comment:
can we just remove the items we childs units we don't want to add instead of adding additional logic?